### PR TITLE
chore: update gitpod config

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
- update use of `await-port` to `ports await`
- update extension versions

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Comment by Mashy00110011** [comment]
فاااتم الإرسال من هاتف Huawei الخاص بي

**Comment by Mashy00110011** [comment]
تم الإرسال من هاتف Huawei الخاص بي-------- الرسالة الأصلية --------من:‏ "codesee-maps[bot]" ***@***.***>التاريخ:‏ الاثنين، ١٩ سبتمبر، ٢٠٢٢ ١١:٥٥ مإلى:‏ freeCodeCamp/freeCodeCamp ***@***.***>نسخة إلى:‏ Subscribed ***@***.***>الموضوع:‏ Re: [freeCodeCamp/freeCodeCamp] chore: update gitpod config (PR #47566)

👀 Review this PR in a CodeSee Review Map



  View the CodeSee Map of this change



Legend

File: .gitpod.yml [hunk 1]
```diff
@@ -45,13 +45,13 @@ tasks:
       cp sample.env .env &&
       npm ci &&
       gp sync-done npm-ci &&
-      gp await-port 27017 &&
+      gp ports await 27017 &&
```
Comment (naomi-lgbt) [inline .gitpod.yml:48]: I don't use GitPod - what does this change achieve?

**Review by naomi-lgbt — APPROVED** [review]
You use this as your daily driver, so I'm gonna trust you didn't break it 😉

File: .gitpod.yml (hunk 1 above)
Comment (ShaunSHamilton) [reply .gitpod.yml:48]: `await-port` is deprecated.

**Review by raisedadead — APPROVED** [review]

**Comment by Mashy00110011** [comment]
--
Sent from Mail.ru app for Android

**Pull request merged**